src/components/pages/AdminDashboard/ModerationTools.js

```diff
@@ -80,6 +82,42 @@ const ModerationTools = () => {
     });
   };
 
+  // Get information about API tables
+  const handleGetTableInfo = () => {
+    getTableInfo(1).then(res => {
+      setTableInfo(res.data);
+      console.log(res.data);
+    });
+  };
+
+  // Add read/write/draw submissions for test user
+  const handleGenerateTestUserSubs = () => {
+    generateTestUserSubs(1).then(res => {
+      console.log(res.data);
+    });
+  };
+
+  // Remove read/write/draw submissions for test user
+  const handleResetTestUserSubs = () => {
+    resetTestUserSubs(1).then(res => {
+      console.log(res.data);
+    });
+  };
+
+  // Remove read/write/draw submissions for test user
```

Comment:
My literal only thought would be to maybe differentiate between these functions in the comments, since I wasn't sure initially what removing points had to do with submissions.  Otherwise, everything looks really great to me! I like all of these changes